product-scenarios/4-application-development-using-apis/4.1-application-developers-should-be-able-to-use-apis-in-their-applications/4.1.3-delete-registered-application/src/test/java/org/wso2/am/scenario/tests/delete/registered/application/DeleteRegisteredApplicationNegativeTestCase.java

```diff
@@ -0,0 +1,71 @@
+package org.wso2.am.scenario.tests.delete.registered.application;
```

Comment:
Shall we add license?

Reply:
Fixed with https://github.com/wso2/product-apim/pull/5248